reference-architectures/full-stack/deploy-arch-ibm-pvs-inf-full-stack.md

```diff
@@ -45,19 +45,19 @@ content-type: reference-architecture
 {: toc-use-case="ITServiceManagement"}
 {: toc-compliance="SAPCertified"}
 
-Building upon the VPC service that were previously created when you deployed Secure infrastructure on VPC for regulated industries, Power Infrastructure for deployable architectures creates a Power Virtual Server workspace and connects it with VPC services. Proxy service for public internet access from PowerVS workspace is configured.
+'Power Infrastructure for deployable architecture' in variation 'full-stack' creates in one single deployment Power Virtual Server workspace and connects it with VPC services. Proxy service for public internet access from PowerVS workspace is configured.
```

Comment:
'Power Infrastructure for deployable architecture' in full-stack variation 'PowerVS workspace' creates in one single deployment Power Virtual Server workspace and connects it with VPC services. Proxy service for public internet access from PowerVS workspace is configured.

Reply:
Suggest this to coincide with my suggestion for the extension variation. 

The "full stack" variation of the Power Infrastructure for deployable architectures creates a Power Virtual Server workspace and connects it with VPC services. Proxy service for public internet access from PowerVS workspace is configured.

A proxy service for public internet access from the PowerVS workspace is configured. You can optionally configure some management components on VPC (such as an NFS server, NTP forwarder, and DNS forwarder).